test/fixtures/TestEdition.scala

```diff
@@ -41,6 +41,7 @@ object TestEdition extends RegionalEdition {
   override val editionType: EditionType = EditionType.Regional
   override val notificationUTCOffset: Int = 2
   override val topic: String = "tt"
+  override val locale: Option[String] = Some("tt_TT")
```

Comment:
I doubt if we will ever want the Test Edition, but this isn't a valid Locale.  I'd suggest using some valid one.

Do the tests correctly check this value?